menas/ui/components/schema/schemaMain.controller.js

```diff
@@ -104,14 +104,17 @@ sap.ui.controller("components.schema.schemaMain", {
 	},
 
 	resetNewSchemaValueState : function() {
```

Comment:
Could you please replace the tabs for two spaces in the file to conform to agreed style

Reply:
I know you "inherited" this

Reply:
this method is not  part of the this task.

Reply:
Ok. I have fixed it.

Reply:
I know. But, the whole file is then in mixed mode. Thank you!